cpp/benchmarks/flow_simulation_ode_seir.cpp

```diff
@@ -32,61 +32,92 @@ namespace mio
 namespace benchmark
 {
 
-using FlowModel = ::mio::oseir::Model;
+using FlowModel = ::mio::oseir::Model<ScalarType>;
 
 using namespace oseir;
 
 // For comparison benchmarks, an old model version that does not provide computation of flows has been reimplemented here.
 // For more details see the original implementation in:
 // https://github.com/SciCompMod/memilio/blob/13555a6b23177d2d4633c393903461a27ce5762b/cpp/models/ode_seir/model.h
-class FlowlessModel : public CompartmentalModel<InfectionState, Populations<InfectionState>, Parameters>
+class FlowlessModel
+    : public CompartmentalModel<ScalarType, oseir::InfectionState,
+                                Populations<ScalarType, AgeGroup, oseir::InfectionState>, oseir::Parameters<ScalarType>>
 {
-    using Base = CompartmentalModel<InfectionState, mio::Populations<InfectionState>, Parameters>;
+    using InfectionState = oseir::InfectionState;
+    using Parameters     = oseir::Parameters<>;
```

Comment:
Why do you define these two? These types are inherited from CompartmentalModel. And Parameters<> could be a different type from Parameters<ScalarType>, please only use the latter for now.